research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+---
+menu: Proposals
+name: Tabs (Editor's Draft)
+path: /components/tabs
+pathToResearch: /components/tabs.research
+showInMenu: false
+---
+
+# Tabs Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+*Tabs* are a set of layered sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
```

Comment:
Is it worth mentioning that they are well defined in accessibility trees and long defined as roles and relationships in ARIA?

Reply:
I think it's always worth mentioning, but perhaps there should be an ARIA/a11y section later on?

Reply:
We should at least reference / link to the interaction model below in the a11y section. Added a note there as well.

Reply:
I added this, thanks.